modules/gitops-configure-rollout-route-traffic-using-openshift-service-mesh.adoc

```diff
@@ -0,0 +1,448 @@
+----
+$ oc argo rollouts set image rollouts-demo rollouts-demo=argoproj/rollouts-demo:yellow -n <namespace>
+----
++
+As a result, the container image deployed in the rollout is modified and the rollout initiates a new canary deployment.
++
+[NOTE]
+====
+As per the `setWeight` property defined in the `.spec.strategy.canary.steps` field of the `Rollout` resource, initially 20% of traffic to the route reaches the canary version and 80% of traffic is directed towards the stable version. The rollout is paused after 20% of traffic is directed to the canary version.
+====
+
+.. Watch the progression of your rollout by running the following command:
++
+--
+[source,terminal]
+----
+$ oc argo rollouts get rollout rollouts-demo --watch -n <namespace> <1>
+----
+<1> Specify the namespace where the `Rollout` resource is defined.
+--
++
+In the given example, 80% of traffic is directed towards the stable version and 20% of traffic is routed to the canary service. The deployment is then paused indefinitely until you request for promotion to the next level.
++
+.Example output
+[source,terminal]
+----
+Name:            rollouts-demo
+Namespace:       argo-rollouts
+Status:          ॥ Paused
+Message:         CanaryPauseStep
+Strategy:        Canary
+  Step:          1/8
+  SetWeight:     20
+  ActualWeight:  20
+Images:          argoproj/rollouts-demo:blue (stable)
+                 argoproj/rollouts-demo:yellow (canary)
+Replicas:
+  Desired:       5
+  Current:       6
+  Updated:       1
+  Ready:         6
+  Available:     6
+
+NAME                                       KIND        STATUS     AGE    INFO
+⟳ rollouts-demo                            Rollout     ॥ Paused   6m51s  
+├──# revision:2                                                          
+│  └──⧉ rollouts-demo-6cf78c66c5           ReplicaSet  ✔ Healthy  99s    canary
+│     └──□ rollouts-demo-6cf78c66c5-zrgd4  Pod         ✔ Running  98s    ready:1/1
+└──# revision:1                                                          
+   └──⧉ rollouts-demo-687d76d795           ReplicaSet  ✔ Healthy  9m51s  stable
+      ├──□ rollouts-demo-687d76d795-75k57  Pod         ✔ Running  9m50s  ready:1/1
+      ├──□ rollouts-demo-687d76d795-jsxg8  Pod         ✔ Running  9m50s  ready:1/1
+      ├──□ rollouts-demo-687d76d795-rsgtv  Pod         ✔ Running  9m50s  ready:1/1
+      └──□ rollouts-demo-687d76d795-xrmrj  Pod         ✔ Running  9m50s  ready:1/1
+----
++
+.Example with 80% directed to the stable version and 20% of traffic directed to the canary version.
+[source,yaml]
+----
+route
+- destination:
+    host: rollouts-demo-stable
+  weight: 80 #<1>
+- destination:
+    host: rollouts-demo-canary
+  weight: 20 #<2>
+----
+<1> A value of `80` means that 80% of traffic is directed to the stable version.
+<2> A value of `20` means that 20% of traffic is directed to the canary version.
+
+. Since the deployment is paused, you can let it proceed by manually promoting to the next promotion step.
```

Comment:
Each step needs to have its [first sentence in the imperative mood](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#writing-steps), meaning "you can" should be removed and users should simply be instructed to take the action:
```suggestion
. Since the deployment is paused, let it proceed by manually promoting to the next promotion step.
```

Reply:
Fixed.